package.json

```diff
@@ -36,6 +36,7 @@
   "dependencies": {
     "@notionhq/client": "^2.2.3",
     "core-js": "^3.29.1",
+    "dayjs": "^1.11.8",
```

Comment:
With the other suggested changes, we should be able to remove the dependency on `dayjs`.